signature-aggregator/aggregator/aggregator.go

```diff
@@ -214,7 +215,7 @@ func (s *SignatureAggregator) CreateSignedMessage(
 	reqBytes, err := s.marshalRequest(unsignedMessage, justification, sourceSubnet)
 	if err != nil {
 		msg := "Failed to marshal request bytes"
-		s.logger.Error(
+		log.Error(
```

Comment:
You can get rid of the `zap.String("warpMessageID", unsignedMessage.ID().String()),` line here